test/extended/deployments/deployments.go

```diff
@@ -1554,4 +1554,42 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
 			o.Expect(strings.TrimSpace(rcs.Items[0].Spec.Template.Spec.Containers[0].Image)).NotTo(o.BeEmpty())
 		})
 	})
+
+	g.Describe("when no triggers are defined [Conformance]", func() {
+		dcName := "paused"
+		g.AfterEach(func() {
+			failureTrap(oc, dcName, g.CurrentGinkgoTestDescription().Failed)
+		})
+
+		g.It("should manually make deploy", func() {
+			dc, err := createDeploymentConfig(oc, pausedDeploymentFixture)
+			o.Expect(err).NotTo(o.HaveOccurred())
+			o.Expect(dc.Name).To(o.Equal(dcName))
+
+			g.By("remove all the triggers")
+			_, err = oc.Run("set").Args("triggers", "dc/"+dcName, "--remove-all").Output()
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("resume the DC")
+			_, err = oc.Run("rollout").Args("resume", "dc/"+dcName).Output()
+			o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
Can we choose test data other than pausedDeploymentFixture so we can reduce the "resume" operation in this test?

Reply:
Sorry, If we updated with other test data , we need to add other steps to check the non-auto deployment. So this one is appropriate。

Reply:
OK